packages/react-widget/src/VoucherifyRedeem.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react'
```

Comment:
Try widget configuration without setting `amount` property and you won't be able to redeem voucher at all.